src/Samples/SubModel/Program.fs

```diff
@@ -4,6 +4,41 @@ open System
 open Elmish
 open Elmish.WPF
 
+module Counter =
+
+  type Model =
+    { Count: int
+      StepSize: int }
+  
+  let init () =
+    { Count = 0
+      StepSize = 1 }
+  
+  type Msg =
+    | Increment
+    | Decrement
+    | SetStepSize of int
+    | Reset
+    
+  let canReset = (<>) (init ())
+  
+  let update msg m =
+    match msg with
+    | Increment -> { m with Count = m.Count + m.StepSize }
+    | Decrement -> { m with Count = m.Count - m.StepSize }
+    | SetStepSize x -> { m with StepSize = x }
+    | Reset -> init ()
+  
+  let bindings () : Binding<Model, Msg> list = [
+    "CounterValue" |> Binding.oneWay (fun m -> m.Count)
+    "Increment" |> Binding.cmd Increment
+    "Decrement" |> Binding.cmd Decrement
+    "StepSize" |> Binding.twoWay(
+      (fun m -> float m.StepSize),
+      int >> SetStepSize)
+    "Reset" |> Binding.cmdIf(Reset, (<>) (init ()))
```

Comment:
Shouldn't this use `canReset`? (Unless inlined, of course)

Reply:
Inlined expression